src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -1486,23 +1486,23 @@ OBJECTREF* GcInfoDecoder::GetRegisterSlot(
     _ASSERTE(regNum >= 0 && regNum <= 14);
     _ASSERTE(regNum != 13);  // sp
 
-    DWORD **ppReg;
+    uint32_t **ppReg;
```

Comment:
This needs `#ifdef FEATURE_REDHAWK` similar to other `GcInfoDecoder::GetRegisterSlot` in this file

Reply:
How I miss that. Now it make sense!